.github/CONTRIBUTING.md

```diff
@@ -1,2 +1,63 @@
+
+### Fortran code style standards
+Please see our Fortran [code style guidelines](CODESTYLE.md)
+
+### Universal guidelines
+
+* All code contributions must be made through GitHub following specific guidelines listed below.
+* All contributions must be made in an open, and public way via the GitHub repository. There will be no embargo period for code contributions.
+* All contributions must be able to be merged without conflict. It is the responsibility of the contributor to resolve any merge conflicts **prior** to submitting a Pull Request.
+* All contributions must pass relevant automated testing procedures. These tests are executed automatically when you submit your pull request.
+* Be respectful when giving and receiving feedback on contributions or issues. 
+
+### Pull requests
+
+#### Contributor responsibility
+Pull requests may be denied for a variety of reasons. It is the responsibility of contributors making pull requests to follow the best practices. Not following best practices described below is grounds for rejection of pull requests and may result in excessive manual work for contributors. It is the contributors’ responsibility to know and follow best git practices.
+
+#### Commits: practices & messages
+Learning to use commits wisely and “atomically” is the foundation for success. As users gain familiarity with basics of how git works and the merging practices we are using. The next level of sophistication is interaction with the git log. Reading logs is greatly facilitated by writing good logs, or “commit messages”. The following webpage gives a crash course on how to write good commit messages: https://chris.beams.io/posts/git-commit/
+
+#### Frequency
+Pull requests should happen with high enough frequency to minimize the scope of the code review. Commits which do not changes answers (pass regression testing) MUST be kept separate from commits which do change answers even when the work is on the same feature or bug. The portion of the code which changes the answer must be isolated as much as possible into its own pull request.
+
+### Types of contributions
+#### Bug fixes
+All bug fixes must address a specific [GitHub issue](https://github.com/NCAR/wrf_hydro_nwm_public/issues). Only issues labeled with "Community dev" are open to community contributions. The majority of issues will be open to community contributions, but some cases may require more careful handling. Issues without the "Community dev" label are either still being investigated, are of broad scope, or involve changes to code that is actively being developed internally. 
+
+All code changes to address issues must be submitted using a pull request, and the corresponding issue number must be referenced in the pull request. For more information on referencing issues, see [issues and keywords](https://help.github.com/articles/closing-issues-using-keywords/).
```

Comment:
Key here is going to be to lead by example! This PR and its commits don't reference any issues.

Reply:
Although I agree with you, this repo is still 'just getting off the ground' and there are going to be a number of things that don't follow the best practices in these initial phases